help/include/send-dm.md

```diff
@@ -18,28 +18,34 @@
 
 !!! tip ""
 
-    You can switch from composing a stream message to composing a direct
-    message by selecting **Direct message** from the dropdown in the upper left
-    of the compose box.
+    You can also click on any user in the right sidebar to start composing
+    a direct message to them. Or open their **user card** by clicking on
+    their profile picture or name, and select **Send direct message**.
 
 {tab|mobile}
 
 1. Tap the **Direct messages**
    ( <img src="/static/images/help/mobile-dm-tab-icon.svg" alt="direct messages" class="help-center-icon"/> )
    tab at the bottom of the app.
 
-1. Tap the **New group DM** button at the top of the app.
+1. Tap the **New DM** button at the top of the app to message one person, or
+   **New group DM** to message any number of people.
 
 1. Start typing the name of the person you want to message, and
-   select their name from the list of suggestions. You can continue
-   adding as many message recipients as you like.
+   select their name from the list of suggestions.
 
-1. Approve by tapping the **checkmark**
-   (<img src="/static/images/help/mobile-check-circle-icon.svg" alt="checkmark" class="help-center-icon"/>)
-   button in the bottom right corner of the app.
+1. *(if sending a group DM)* Continue adding as many message recipients as you
```

Comment:
```suggestion
1. *(If sending a group DM)* Continue adding as many message recipients as you
```

Reply:
I'm not sure I agree with this one. At least, no capitalization is more consistent with `git grep "(optional)" help/`.

Reply:
Fair enough :)